### ISSUE TITLE
-0 doesn't do what it looks like

### ISSUE THREAD

**Issue by timmc** [sunng87/shake#3]
(xargs -0 ...) is going to surprise people who don't notice that the negation is going to be stripped by the Clojure reader.

**Comment by sunng87** [comment]
Currently there is a workaround for this:

(xargs "-0" ...)

This issue also affects commands like:

(date "+%s")

**Comment by timmc** [comment]
Yes, the workaround is clear enough, but only if you know to use it. Perhaps your shell macro should reject literal 0 (and therefore -0) with an error message that highlights the ambiguity.
